### DISCUSSION TITLE
Use different Icon (favicon)  for Codespaces

### DISCUSSION THREAD

**Discussion by ewwink** [community/community#113441] [Codespaces]
### Select Topic Area

Product Feedback

### Body

guess where tab for VSCode codespace? 😅

![0](https://github.com/community/community/assets/760764/702a05c7-22ef-4a9a-a90e-5df380a12921)

versus different (color) icon

![02](https://github.com/community/community/assets/760764/953d6ca9-32d8-4772-b658-79fb13a3d4a0)

**Comment by spenserblack** [comment]
Interesting, I use the "dark dimmed" theme, and the codespace and "main" GitHub icons are already different colors :laughing:
I guess this affects light mode users (and maybe "non-dimmed" dark theme users).

**Comment by simchavos** [comment]
Bump, will this be addressed?
